### ISSUE TITLE
Pinned Donations Should Always Be Displayed

### ISSUE THREAD

**Issue by jonathansampson** [brave/browser-laptop#12584]
### Test plan
See https://github.com/brave/browser-laptop/pull/13097

### Description
Pinned-donations may not be revealed in the Brave Payments site-list if they don't meet other criteria.

### Steps to Reproduce
1. Set the threshold to 5 visits, and quality a few domains
2. Pin one of the domains
3. Increase the threshold to 10 visits

**Actual result:**
Pinned donations are no longer shown if they don't meet the _visits_ requirement.

**Expected result:**
Pinned donations should _always_ be shown.

**Reproduces how often:**
100%

### Brave Version
0.19.131

**about:brave info:**
```
Brave: 0.19.131 
rev: da2f34787e51eb44190a7a77249e100fb2368306 
Muon: 4.5.36 
libchromiumcontent: 63.0.3239.132 
V8: 6.3.292.49 
Node.js: 7.9.0 
Update Channel: Release 
OS Platform: Microsoft Windows 
OS Release: 10.0.16299 
OS Architecture: x64
```

**Reproducible on current live release:**
Yes

**Comment by NejcZdovc** [comment]
closing as #13097 was merged

**Comment by LaurenWags** [comment]
Removed QA/checked labels due to changes in milestones. Please recheck with 0.21.15 and higher.

**Comment by srirambv** [comment]
Broken on 0.21.16
![13097](https://user-images.githubusercontent.com/17010094/36727113-0f54ebd2-1be2-11e8-8af1-e3e04d7d8340.gif)

**Comment by NejcZdovc** [comment]
it's not broken, it was not merged in this 0.21 milestone. PR and issues have different milestones. Will merge it now.

**Comment by NejcZdovc** [comment]
PR merged into 0.21

**Comment by srirambv** [comment]
Works on 0.21.17 
![12584](https://user-images.githubusercontent.com/17010094/36780076-ee80ab4c-1c97-11e8-9292-53790d18dd6f.gif)
